lib/forms/pegasus_form_validation.rb

```diff
@@ -4,6 +4,7 @@
 require_relative 'pegasus_form_errors'
 require_relative '../cdo/regexp'
 require_relative '../cdo/geocoder'
+require_relative '../../pegasus/forms/hoc_signup_2017'
```

Comment:
On that note, I'd be ok if the validator required the entire `forms` directory. I just don't think it should know or care about any specific form.